allennlp/modules/seq2seq_encoders/__init__.py

```diff
@@ -4,26 +4,26 @@
 
 from allennlp.common import Params
 from allennlp.common.checks import ConfigurationError
-from allennlp.experiments import Registry
+from allennlp.modules import Seq2SeqEncoder
 from allennlp.modules.seq2seq_encoders.pytorch_seq2seq_wrapper import PytorchSeq2SeqWrapper
 
 class _Seq2SeqWrapper:
     """
-    The :class:`Registry` needs to have a ``Type[Seq2SeqEncoder]`` as the value registered for each
```

Comment:
@DeNeutoy, I'm guessing we can simplify this quite a bit, just handling this specially `Seq2SeqEncoder.from_params()`, now that we're not going through the registry.  We can definitely worry about that later, in another PR, though.